doc/source/whatsnew/v0.25.0.rst

```diff
@@ -19,6 +19,7 @@ including other versions of pandas.
 Other Enhancements
 ^^^^^^^^^^^^^^^^^^
 
+- Indexing and __getitem__ of DataFrame and Series now accept zerodim np.ndarray. (:issue:`24919`)
```

Comment:
use double backticks on DataFrame, Series, np.ndarray., say

you don't need to say ``__getitem__`` as Indexing is pretty general here.